i2/AMPLE_gui.py

```diff
@@ -148,6 +113,15 @@ def drawContents(self):
         self.closeSubFrame()
         
         self.drawOptions()
+        
+    def setContactFileOptions(self):
+        import conkit.io
+        parsers = sorted(conkit.io.CONTACT_FILE_PARSERS.keys())
```

Comment:
The ConKit mask is now in `ample/util/contact_util.py` and should be imported from there.
```
>>> from ample.util.contact_util import CONTACT_FILE_PARSERS
>>> print(CONTACT_FILE_PARSERS)
['bbcontacts', 'bclcontact', 'casp', 'ccmpred', 'comsat', 'epcmap', 
 'evfold', 'freecontact', 'gremlin', 'membrain', 'metapsicov', 'mmcif', 
 'pconsc', 'pconsc2', 'pconsc3', 'pdb', 'plmdca', 'psicov']
```
The default, as mentioned above, should be set to `casp`.